core/src/main/java/org/apache/accumulo/core/spi/compaction/CompactionPlanner.java

```diff
@@ -79,6 +81,8 @@ public interface InitParameters {
    */
   public interface PlanningParameters {
 
+    NamespaceId getNamespaceId() throws TableNotFoundException;
```

Comment:
This is SPI code so should add a since tag.

```suggestion
   /**
     * @since 2.1.4
     */
    NamespaceId getNamespaceId() throws TableNotFoundException;
```